src/renderer/account/components/AccountPanel/index.js

```diff
@@ -10,6 +10,25 @@ import withNetworkData from 'shared/hocs/withNetworkData';
 
 import AccountPanel from './AccountPanel';
 
+const fetch = require('isomorphic-fetch');
```

Comment:
Can we just use `node-fetch` here instead of adding another dependency to package.json?

Reply:
thank you, I didn't noticed node-fetch